index.html

```diff
@@ -57,6 +57,11 @@
         SwaggerEditorStandalonePreset
       ],
       queryConfigEnabled: false,
+    },
+    {
```

Comment:
Correct me if I'm wrong, but this looks line unnecessary change. The configuration options are already present and have their default values set:

https://github.com/swagger-api/swagger-editor/blob/2581d92dcf984bea0b6c19d96894f5f24dd1f03a/src/index.js#L63

Reply:
You're right, see my answer: https://github.com/swagger-api/swagger-editor/pull/2910#discussion_r821852314